projects/aws-observability/aws-otel-collector/helm/patches/0003-Update-default-values.patch

```diff
@@ -1,27 +1,14 @@
-From e90f72f6a1aa4a0948433b165d06f33757d6d88e Mon Sep 17 00:00:00 2001
-From: ahreehong <46465244+ahreehong@users.noreply.github.com>
-Date: Thu, 18 Jan 2024 14:38:34 -0800
-Subject: [PATCH] Update default values
+From 4f976750a86fbee6cf4683f55f071a9c4bce1dbd Mon Sep 17 00:00:00 2001
+From: Prow Bot <prow@amazonaws.com>
```

Comment:
Curious why is this Prow Bot? Is this coming from local git config or generated by a presubmit job?

Reply:
Yes this is from local git config, during the package build, package is cloned by Prow Bot.
So it got changed during resolution of merge conflict when patch was recreated.

Reply:
Why would it be cloned by Prow Bot if local git config? I am confused by that, unless our make scripts are setting it which I feel it shouldn't?

Reply:
I think you were just trying to resolve merge conflicts instead of rerunning the build, that's what you are saying right?